packages/@cdktf/hcl2cdk/test/__snapshots__/jsiiLanguage.test.ts.snap

```diff
@@ -285,7 +285,7 @@ class MyConvertedCode extends cdktf.TerraformStack {
     new aws.s3BucketObject.S3BucketObject(this, \\"examplebucket_object\\", {
       bucket: \`\\\\\${element(\${awsS3BucketExamplebucket.fqn}, 0).id}\`,
       key: \\"someobject\\",
-      kmsKeyId: awsKmsKeyExamplekms.arn,
+      kmsKeyId: cdktf.Token.asString(awsKmsKeyExamplekms.arn),
```

Comment:
same as above, this seems like an unwanted side effect 🤔

Reply:
Yes, but it's okay, it's likely related to the order of execution slightly changing